craft_providers/lxd/launcher.py

```diff
@@ -92,6 +92,15 @@ def _create_instance(
             project=project,
         )
 
+    # set the full instance name as image description
+    lxc.config_set(
+        instance_name=instance.instance_name,
```

Comment:
Should this be `base_instance.instance_name`?

Reply:
Yes, your are correct.